osrf/vrx
# Follow-the-path task

The follow the path task including:

* Example world
* Practice worlds
* We can reuse the navigation scoring plugin

How to test it?

1. Launch the simulation:
```
ros2 launch vrx_gz competition.launch.py world:=follow_path_task
```

2. Launch the joystick teleoperation node:
```
ros2 launch vrx_gz usv_joy_teleop.py
```

3. Teleoperate the robot through the course. The task should successfully finish after crossing the last gate. Crossing the gates in the wrong error should also trigger the task to finish (unsuccessfully).
